commit c140151eba0e3fc6bf3696b428d1b9130d99c40e
Author: Mark Hargreaves <clashclanacc2602@gmail.com>
Date:   Thu Aug 6 13:23:03 2020 +0300

    Revert "wlroots: Fix android headers path"
    
    This reverts commit 3337f551b06ee38a458e9259a34fcb166e58fef7.

diff --git a/meson.build b/meson.build
index 83ef5025..374ee288 100644
--- a/meson.build
+++ b/meson.build
@@ -148,7 +148,7 @@ add_project_arguments([
 	'-DLIBINPUT_MAJOR=' + libinput_ver[0],
 	'-DLIBINPUT_MINOR=' + libinput_ver[1],
 	'-DLIBINPUT_PATCH=' + libinput_ver[2],
-	'-I/usr/include/android',
+	'-I/usr/include/android-9.0',
 ], language: 'c')
 
 subdir('protocol')
